docs/reference/extensions/django-framework.rst

```diff
@@ -0,0 +1,35 @@
+.. _django-framework-reference:
+
+django-framework
+----------------
+
+The Django extension streamlines the process of building Django application
+rocks.
+
+It facilitates the installation of Django application dependencies, including
+Gunicorn, inside the rock. Additionally, it transfers your project files to
+``/django/app`` within the rock.
+
+A statsd-exporter is installed alongside the Gunicorn server to export Gunicorn
+server metrics.
+
+.. note::
+    The Django extension is compatible with the ``bare`` and ``ubuntu@22.04``
+    bases.
+
+Project requirements
+====================
+
+There are 2 requirements to be able to use the ``django-framework`` extension:
+
+1. There must be a ``requirements.txt`` file in the root of the project with
```

Comment:
Would it be clearer to say "root directory"?

Reply:
Fixed